lib/kubeclient/common.rb

```diff
@@ -416,6 +428,18 @@ def api
       JSON.parse(response)
     end
 
+    def self.restclient_read_timeout_option
```

Comment:
couldn't private because used by test too.
but yeah, this actually exposes it, not nice :-(

- use `send` to bypass private in test?
- move to separate class (Kubeclient::RestClientCompatibility)?